src/libs/tracer/debugTraceCall.ts

```diff
@@ -71,8 +73,29 @@ export async function debugTraceCall(
       },
       'latest',
       {
-        tracer:
-          "{data: [], fault: function (log) {}, step: function (log) { if (log.op.toString() === 'LOG3') { this.data.push([ toHex(log.contract.getAddress()), '0x' + ('0000000000000000000000000000000000000000' + log.stack.peek(4).toString(16)).slice(-40)])}}, result: function () { return this.data }}",
+        tracer: `{
+          data: [], 
+          fault: function (log) {}, 
+          step: function (log) { 
+            if (log.op.toString() === 'LOG3') { 
+              this.data.push(toHex({
+                type:"ERC-20",
```

Comment:
we mix quote styles here, i would probably do `erc: 20` instead of `type: "ERC-20"` though. because all types would be ERCs